README.md

```diff
@@ -426,6 +426,29 @@ There's also a helper for generating image tags:
 <%= attachment_image_tag(@user, :profile_image, :fill, 300, 300) %>
 ```
 
+You can also provide a limit to the image:
+
+``` erb
+<%= link_to "Image", attachment_url(@user, :profile_image, :limit, 400, 500) %>
+```
+
+If you don't care about the aspect ration and want an exact dimension, you can use `!`:
+
+``` erb
+<%= link_to "Image", attachment_url(@user, :profile_image, :limit, 400, 1000!) %>
```

Comment:
You probably wanted to use `"1000!"` instead of `1000!`.

Reply:
Oh, thank you!